docs/.vitepress/theme/components/landing/3. frameworks-section/FrameworksSection.vue

```diff
@@ -164,6 +165,13 @@ const frameworks: Framework[] = [
     url: 'https://emberjs.com/',
     visible: ref(false),
   },
+  {
+    name: 'HonoJS',
```

Comment:
It looks like hono usually refers itself as only "hono". maybe we can shorten this?

```suggestion
    name: 'Hono',
```

Reply:
I had overlooked the naming pattern.